Unaccountable-Magpie/BangazonAPI2
# Documentation for EmployeeTrainingPrograms model

Purpose:
Added documentation for author and purpose of the EmployeeTrainingProgram model


How it fits in context of the project:
Clarifies what this model contains at the top of the file


Specific Feature affected:
EmployeeTrainingProgram model contains commented out documentation; no code affected


How to test (Be thorough!):

1. check into branch LR-EmpTrainModelDocumentation

2. open EmployeeTrainingPrograms.cs

3. make sure the documentation at the top is there and commented out
